Ginger/GingerCoreCommon/Repository/RepositoryFolder.cs

```diff
@@ -553,9 +553,21 @@ private void HandleDirecortyChange(FileSystemEventArgs e)
                     break;
                 case WatcherChangeTypes.Changed:
                     // change happen when new file is added, for now we can ignore, as we will get also file added event
+
+                    //if the folder is deleted(shift + del) by the user from the file system and not from Ginger
+                    try
+                    {
+                        Directory.GetFiles(e.FullPath);
+                    }
+                    catch (Exception ex)
+                    {
+                        RepositoryFolder<T> rf = GetSubFolder(fn);
+                        rf.StopFileWatcherRecursive();
+                    }
                     break;
                 case WatcherChangeTypes.Deleted:
                     RepositoryFolder<T> sf2 = GetSubFolder(fn);
+                    //sf2.StopFileWatcherRecursive();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5863082)